sequencing/clinical_annotator/dxapp.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "clinical_annotator",
+  "title": "Clinical Variant Annotator",
+  "summary": "Annotates a VCF File with most recent ClinVar TSV file and with a user provided HGMD PRO VCF file.",
+  "dxapi": "1.0.0",
+  "version": "1.0.0",
+  "inputSpec": [
+    {
+      "name": "variants_vcfgz",
+      "label": "VCF File(s)",
+      "class": "array:file",
+      "optional": false,
+      "patterns" : ["*.vcf.gz"],
+        "help": "The VCF File to be annotated (chr prefix must not be present in VCF)."
+    },
+    {
+      "name": "variant_summary",
+      "label": "Variant Summary File",
+      "class": "file",
+      "optional": false,
+      "patterns": ["*.txt.gz"],
+      "help": "A text file containing summarization of variant annotations (one such file can be downloaded here: ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz)."
+    },
+    {
+      "name": "variants_vcfgztbi",
+      "label": "VCF File Indices",
+      "class": "array:file",
+      "optional": true,
+      "patterns": [
+        "*.vcf.gz.tbi"
+      ],
+        "help": "(Optional) Tabix index file for VCF to be annotated."
+    },
+    {
+      "name": "hgmd_pro_file",
+      "label": "HGMD PRO File",
+      "class": "file",
+      "optional": true,
+      "patterns": [
+        "*.vcf","*.vcf.gz"
+      ],
+      "help": "(Optional) Make sure the build version matches the VCF to be annotated (infomation on purchasing an HGMD PRO license can be found at: https://www.qiagenbioinformatics.com/products/human-gene-mutation-database/)."
+    },
+    {
+      "group": "Build Options",
+      "name": "build_version",
+      "label": "NCBI Build Version",
+      "class": "string",
+      "choices": ["b38","b37"],
+      "default": "b38",
+      "optional": false,
+      "help": "Please indicate which build your VCF is (the default is NCBI b38)."
+    }
+  ],
+  "outputSpec": [
+    {
+      "name": "out_variants_vcfgz",
+      "label": "VCF File(s)",
+      "class": "array:file",
+      "patterns" : ["*.vcf.gz"],
+      "help": "VCF file(s) with included annotations."
+    },
+    {
+      "name": "out_variants_vcfgztbi",
```

Comment:
"out_variants_vcfgztbi" -> "out_variants_vcfgztbis"
Also need to update such input variable name in the app script.